common/v2/features/SendAssets/components/SendAssetsForm.tsx

```diff
@@ -302,23 +300,23 @@ export default function SendAssetsForm({ txConfig, onComplete }: IStepComponentP
             }
           };
 
-          const handleDomainResolve = async (name: string) => {
+          const handleDomainResolve = async (name: string): Promise<string | undefined> => {
             if (!values || !values.network) {
               setIsResolvingDomain(false);
               setResolutionError(undefined);
               return;
             }
             setIsResolvingDomain(true);
             setResolutionError(undefined);
+            const asset =
+              getBaseAssetByNetwork({ network: values.network, assets: userAssets }) ||
+              values.asset;
```

Comment:
As far as I know, ENS is per network (ETH, ROPSTEN, ETC...), not per asset (BAT, AAC...) that is why `baseAsset` is used. I see that we already have `baseAsset` in component's state that is why I used it instead of finding it again.